distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex/Pattern.enso

```diff
@@ -2,9 +2,9 @@ import project.Any.Any
 import project.Data.Filter_Condition.Filter_Condition
 import project.Data.Map.Map
 import project.Data.Numbers.Integer
-import project.Data.Range.Extensions
+import project.Data.Range.Range_Extensions
 import project.Data.Range.Range
-import project.Data.Text.Extensions
+import project.Data.Text.Text_Extensions
```

Comment:
`Range_Extensions`, `Text_Extensions` and other `_Extensions`... I am fine with that - however the final call isn't on me.

It would be nicer if we could just import `Range` and then always type `Range.Extensions` - but I guess we can't do it as `import Xyz_Extensions` has the _"side effect"_ of bringing into the _module scope_ all the extension methods defined in the imported module. That's not something we could achieve with `Range.Extensions`.

Unless `from project.Data.Range.Extensions import all` would bring the method in! Doesn't it bring the extension methods in? If so, I'd be in favor of keeping `Xyz.Extensions` and loading them in via `from ... import all`.

Reply:
Right. I tried doing, e.g., `from Standard.Base.Data.Text.Extensions import all`, and it seems to import all the extension methods. I will revert this "Extensions module renaming" then.